sklearn/svm/_classes.py

```diff
@@ -177,9 +178,9 @@ class LinearSVC(BaseEstimator, LinearClassifierMixin,
     >>> print(clf.predict([[0, 0, 0, 0]]))
     [1]
     """
-
-    def __init__(self, penalty='l2', loss='squared_hinge', dual=True, tol=1e-4,
-                 C=1.0, multi_class='ovr', fit_intercept=True,
+    @_deprecate_positional_args
+    def __init__(self, penalty='l2', *, loss='squared_hinge', dual=True,
```

Comment:
any reason to make penalty positional but not the loss?
I feel like both are equally unambiguous given the possible names